cmd/justgo/justgo.go

```diff
@@ -51,7 +62,20 @@ func buildProject(path string) {
 
 	fileUrl := "https://github.com/inadarei/justgo/archive/master.zip"
 	tmpFilePath := os.TempDir() + "justgo.zip"
-	//fmt.Println("Downloading to ", tmpFilePath)
+
+	// Move all conflicting files to tmp dir and move them back post-build
+	filesToMove := getConflictingFiles(path)
+	if filesToMove != nil {
+		for _, file := range filesToMove {
+			srcPath := filepath.Join(path, file)
+			tmpFilePath := filepath.Join(os.TempDir(), file)
```

Comment:
By using os.TempDir() directly, you will destroy any pre-existing files there which may have the same name(s) as the files you are stashing away. In extreme scenarios - you may run into even more severe conflicting cases.

Rather, let's make sure we create a unique folder under os.TempDir() and do our dirty work there.